synapse/lib/stormtypes.py

```diff
@@ -953,8 +953,13 @@ def ctordata(path=None):
     async def _methNodeNdef(self):
         return self.valu.ndef
 
-    async def _methNodeRepr(self, name=None):
-        return self.valu.repr(name=name)
+    async def _methNodeRepr(self, name=None, defv=None):
+
+        try:
+            return self.valu.repr(name=name)
+
+        except s_exc.NoPropValu:
+            return defv
```

Comment:
I'm not super thrilled with always returning defv here if the user hasn't provided a defv.